docs/model/BR.html

```diff
@@ -5442,6 +5442,9 @@ <h4>Parsing:</h4>
 
 </div>
 
+<h4>Comparison Stopwords:</h4>
+<code>Ponto de referência:</code>
```

Comment:
Maybe drop the `:` from the stop word?

Reply:
As discussed. I included this to have literally the same as in the formatting.